.dockerignore

```diff
@@ -1,3 +1,49 @@
-node_modules/*
 .git
-Dockerfile
\ No newline at end of file
+.idea
+Dockerfile
+docker-compose.yml
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.coverage
+.tox
+coverage/
+
+.cache
+
+*.log
+eq.log.*
+
+# Virtualenvs
+env
+env*
+venv
+
+# intellij
+.idea/
+*.ipr
+*.iml
+*.iws
+
+.DS_Store
+
+node_modules/
+app/static/
+tests/results
+tests/coverage
+tests/a11y
+
+app/styles/base/_sprite.scss
```

Comment:
It was in the .gitignore so i assumed it was something that was generated by a script?